portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -49,14 +54,43 @@ public String toString() {
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         String name = request.getParameter("name");
         String comment = request.getParameter("comment");
-        Comment entry = new Comment(name, comment);
 
-        allComments.add(entry);
+        Entity taskEntity = new Entity("Task");
+        taskEntity.setProperty("name", name);
+        taskEntity.setProperty("comment", comment);
+
+        datastore.put(taskEntity);
         response.sendRedirect("/index.html");
     }
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String input = request.getParameter("comment-number");
```

Comment:
nit: I think most readers would interpret "comment-number" as "the number of the comment". perhaps num-comments or max-comments might be clearer?

Reply:
Done. Changed name to "max-comments".